mkt/extensions/models.py

```diff
@@ -267,6 +267,7 @@ def mini_manifest(self):
         except ExtensionVersion.DoesNotExist:
             return {}
         mini_manifest = {
+            'id': self.uuid,
```

Comment:
I chose to expose it as `id` to be coherent with signing (which *needs* it to be called `id`). I'm open to comments about why that's a terrible idea.